app/controllers/application.js

```diff
@@ -213,6 +213,10 @@ export default Controller.extend(
 			 */
 			toggleUserMenu(visible) {
 				this.set('userMenuVisible', visible);
+			},
+
+			toggleSiteHeadShadow(visible) {
+				this.set('siteHeadNoShadow', !visible);
```

Comment:
I think this negation makes the logic harder to follow and is unnecessary. How about `this.set('siteHeadShadow', visible);` here and `<nav class="site-head{{unless shadow ' no-shadow'}}">` in `site-head.hbs`?